bears/typescript/TSLintBear.py

```diff
@@ -17,15 +18,37 @@ class TSLintBear:
     """
 
     LANGUAGES = {"TypeScript"}
-    REQUIREMENTS = {NpmRequirement('tslint', '3')}
+    REQUIREMENTS = {NpmRequirement('tslint', '3'),
+                    NpmRequirement('codelyzer', '*')}
```

Comment:
if latest version is wanted, i think you can specify no version and it will be automatically :D